guide/getting-started/visual-language.md

```diff
@@ -133,6 +134,25 @@ The Unicode symbol for Bitcoin (₿) was introduced in [Unicode 10.0.0](http://w
 
 </div>
 
+## The Satoshi symbol
+
+As the price of bitcoin has increased, and the Lightning network has facilitated very small payments (more on that below), it has become common for people to denominate bitcoin with the "satoshi". See the [Units & Symbols](https://bitcoin.design/guide/payments/units-and-symbols/) section for more detailed information about bitcoin units.
```

Comment:
>  the Lightning network has facilitated very small payments

Not sure what this is meant to refer to? The volume, the  size of the payments (micropayments) or?

Reply:
```suggestion
As the price of bitcoin has increased, and the Lightning network has facilitated low-value payments (more on that below), it has become common for people to denominate bitcoin with the "satoshi". See the [Units & Symbols](https://bitcoin.design/guide/payments/units-and-symbols/) section for more detailed information about bitcoin units.
```

Reply:
I intended to mean micropayments, as in small number of satoshis. I think Christoph's change reflects this well.